front/app/containers/Admin/ideas/all/index.tsx

```diff
@@ -19,35 +19,32 @@ const IdeasTab = () => {
     'topics',
     'statuses',
   ];
-  const { data: projects } = useProjects({
+  const { data: projects, isLoading } = useProjects({
     pageSize: 250,
     sort: 'new',
     publicationStatuses,
     canModerate: true,
   });
 
-  if (projects) {
-    return (
-      <>
-        <Title color="primary">
-          <FormattedMessage {...messages.inputManagerPageTitle} />
-        </Title>
-        <SectionDescription>
-          <FormattedMessage {...messages.inputManagerPageSubtitle} />
-        </SectionDescription>
-        <Box background={colors.white} p="40px">
-          <PostManager
-            type="AllIdeas"
-            defaultFilterMenu={defaultFilterMenu}
-            visibleFilterMenus={visibleFilterMenus}
-            projects={projects.data}
-          />
-        </Box>
-      </>
-    );
-  }
-
-  return null;
+  return (
+    <>
+      <Title color="primary">
+        <FormattedMessage {...messages.inputManagerPageTitle} />
+      </Title>
+      <SectionDescription>
+        <FormattedMessage {...messages.inputManagerPageSubtitle} />
+      </SectionDescription>
+      <Box background={colors.white} p="40px">
+        {isLoading && <Spinner />}
+        <PostManager
+          type="AllIdeas"
+          defaultFilterMenu={defaultFilterMenu}
+          visibleFilterMenus={visibleFilterMenus}
+          projects={projects?.data}
+        />
```

Comment:
```suggestion
        {isLoading ? (
          <Spinner />
        ) : (
          <PostManager
            type="AllIdeas"
            defaultFilterMenu={defaultFilterMenu}
            visibleFilterMenus={visibleFilterMenus}
            projects={projects?.data}
          />
        )}
```

This looks better imo: you won't have the spinner in some weird spot at the top while the post manager is already starting the show.

Reply:
The intention was to show the post manager with the spinner since it might load for a while on some platforms. I have no strong opinions but just showing a spinner that takes a while might not be too good either

Reply:
Okay, checking another option. Don't merge yet :-)

Reply:
@EdwinKato I see two better options:

-Low effort: Wrap Loading with Box that has mb of 28px
-Move the spinner to UI part for which we are showing the spinner: `front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarProjects.tsx`

Reply:
If you have already done it, you can push it and we avoid double work

Reply:
OK, pushed the low-effort solution. If you're up for it, and it doesn't take a lot of time, the second option would still be better.